bittensor/_subtensor/__init__.py

```diff
@@ -134,11 +138,11 @@ def __new__(
         # make sure it's wss:// or ws://
         # If it's bellagene (parachain testnet) then it has to be wss
         endpoint_url: str = config.subtensor.chain_endpoint
-        if endpoint_url[0:6] != "wss://" and endpoint_url[0:5] != "ws://":
-            if config.subtensor.network == "bellagene":
-                endpoint_url = "wss://{}".format(endpoint_url)
-            else:
-                endpoint_url = "ws://{}".format(endpoint_url)
+        if config.subtensor.network == "bellagene":
```

Comment:
should `bellagene` be a constant?

Reply:
this is the name of the new network that @shibshib added